main.go

```diff
@@ -248,38 +251,56 @@ func main() {
 			failf("Failed to open xcproj - (%s), error:", cfg.ProjectPath, err)
 		}
 
-		schemeBuildDir, err := buildTargetDirForScheme(proj, cfg.ProjectPath, cfg.Scheme, cfg.Configuration, cfg.SimulatorPlatform)
+		var customOptions []string
+		for _, customOption := range strings.Split(cfg.XcodebuildOptions, " ") {
+			if customOption != "" {
+				customOptions = append(customOptions, customOption)
+			}
+		}
+
+		// Get the simulator name
+		{
+			simulatorName := iOSSimName
+			if cfg.SimulatorPlatform == "tvOS" {
+				simulatorName = tvOSSimName
+			}
+
+			customOptions = append(customOptions, "-sdk")
+			customOptions = append(customOptions, simulatorName)
+		}
+
+		schemeBuildDir, err := buildTargetDirForScheme(proj, cfg.ProjectPath, cfg.Scheme, cfg.Configuration, customOptions...)
 		if err != nil {
 			failf("Failed to get scheme (%s) build target dir, error: %s", err)
 		}
 
 		// Export the artifact from the build dir to the output_dir
 		if exportedArtifacts, err = exportArtifacts(proj, cfg.Scheme, schemeBuildDir, cfg.Configuration, cfg.SimulatorPlatform, absOutputDir); err != nil {
-			failf("Failed to export the artifacts, error: %s", err)
+			log.Warnf("Failed to export the artifacts, error: %s", err)
```

Comment:
this is the point of the step, if it is not able to build / find the built artifacts it should fail